gnoblog-cli/cmd/post.go

```diff
@@ -0,0 +1,261 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/gnolang/gno/gno.land/pkg/gnoclient"
+	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"os"
+	"strings"
```

Comment:
Your IDE can automatically sort the imports on save. Otherwise, you need to manually run gofumpt.